apps/studio/components/interfaces/Home/Connect/ConnectDropdown.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, ChevronDown } from 'lucide-react'
+import ConnectionIcon from './ConnectionIcon'
+import {
+  Button,
+  CommandItem_Shadcn_,
+  CommandList_Shadcn_,
+  Command_Shadcn_,
+  PopoverContent_Shadcn_,
+  PopoverTrigger_Shadcn_,
+  Popover_Shadcn_,
+} from 'ui'
+
+interface ConnectDropdownProps {
+  level?: 'parent' | 'child' | 'grandchild'
+  open: boolean
+  setOpen: (open: boolean) => void
+  state: string
+  updateState: (state: string) => void
+  label: string
+  items: any[]
+}
+
+const ConnectDropdown = ({
+  level,
+  state,
+  updateState,
+  label,
+  open,
+  setOpen,
+  items,
+}: ConnectDropdownProps) => {
+  function onSelectLib(key: string) {
+    updateState(key)
+    setOpen(false)
+  }
+
+  const selectedItem = items.find((item) => item.key === state)
+
+  return (
+    <>
+      <Popover_Shadcn_ open={open} onOpenChange={setOpen} modal={false}>
+        <div className="flex items-center">
+          <span className="flex items-center text-foreground-muted bg-button px-3 rounded-md rounded-r-none text-xs h-[26px] border border-r-0">
+            {label}
+          </span>
+          <PopoverTrigger_Shadcn_ asChild>
+            <Button size="tiny" type="default" className="h-[26px] pr-3 gap-0 rounded-l-none">
+              <div className="flex items-center gap-1">
```

Comment:
add classes direct to Button

Reply:
ah sorry, think the Button is setup super weird, ignore this.
new Button component won't need this.